### ISSUE TITLE
Vets-api terminal access for [Mitch Saltykov]

### ISSUE THREAD

**Issue by Mitch-A6** [department-of-veterans-affairs/va.gov-team#77721]
### Your Name

Mitch Saltykov

### Your GitHub handle (terminal access is managed through GitHub)

@Mitch-A6

### Team, Role, and Company of the target individual

10-10 Health Apps, Senior Full Stack Engineer, Agile 6

### Product Manager (PM) name and email

Alex Seelig

### Product Owner (PO) name and email

Patrick Bateman

### COR Name (leave "None" if you listed an AWS user name above)

Other - please specify in 'Additional Notes'

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

Agile Six - Anthony M. Arashiro

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [ ] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [X] dev
- [X] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

I need this access to debug issues in 10-10 EZ / CG / EZR forms across all environments.

COR - Pamela Jones, PamelaA.jones@va.gov

PSAC E-QIP  transmissal email screenshot attached:
[
<img width="879" alt="Screenshot 2024-03-05 at 5 27 29 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/153661030/7d3b8443-b49b-445b-85fb-93cf859ef371">
](url)

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by bdech** [comment]
confirmed in Atlas

confirmed E-QIP

**Comment by bdech** [comment]
@Mitch-A6  You’ve been granted access to the `dev`, `staging`, and `sandbox` environments.

Here's some additional documentation for [Vets API on EKS](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-TerminalAccess) that may help.

**Comment by bdech** [comment]
Access to `prod` is pending approval by OCTODE, and we'll update you as soon as it's available.

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by bdech** [comment]
@Mitch-A6 You've been granted access to `Production`, 
with access to Production limited to 6 months, expiring 09/7/24.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1709823534777529)

**Comment by bdech** [comment]
@Mitch-A6 Your Vets-API Terminal Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

**Comment by Mitch-A6** [comment]
I would like to request an extension. I frequently need to use Argo's
access to the Vets API Sidekiq terminal to debug production issues on a
daily-to-weekly basis, and will continue to need to do so for the
foreseeable future.

Thanks, Brandon
--Mitch

**Comment by bdech** [comment]
@LindseySaari @ericboehs can one of you please give us a 👍 to extend their access?

**Comment by bdech** [comment]
@Mitch-A6 your Production access has been extended for 6 months, expiring 03/07/25.

Please set a reminder for yourself to return to this Issue and request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by bdech** [comment]
support slack reminder [here](https://dsva.slack.com/archives/C06CY71PCAX/p1725995194587419)

**Comment by GWright321** [comment]
@Mitch-A6 Your Production access has expired.

Would you like to request an extension? You have 72 hours to respond, or your
access will be revoked and you will need to fill out a new request.

**Comment by Mitch-A6** [comment]
Yes, please give me an extension on my Production access - I still require it on a regular basis to debug prod issues.

**Comment by davidpetras** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by davidpetras** [comment]
@Mitch-A6  your Production access has been extended for 6 months,
expiring 09/11/25.

Please set a reminder for yourself to return to this Issue, reopen it, and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.
